pages/apply/agree-terms.tsx

```diff
@@ -24,7 +24,7 @@ const ApplicationTermsPage = (): JSX.Element => {
   };
 
   return (
-    <Layout pageName="Agreement">
+    <Layout pageName="Agreement" dataTestId="agree-terms-page">
```

Comment:
semantic but we have generally prefixed dataTestId with test - may be worthwhile for consistency at least.

Reply:
yes, definitely. Updated in https://github.com/LBHackney-IT/lbh-housing-register/pull/421/commits/0f9db61943508002de2b659ea834336a9978955c